src/ResourceManager/Compute.ManagedService/Commands.Compute.ManagedService/packages.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Azure.Management.Compute" version="17.0.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="15.1.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Resources" version="2.20.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Storage" version="4.1.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
```

Comment:
The clientruntime and json dependecies should come from ResourceManager.Common

Reply:
ok